pyspark/bigdl/optim/optimizer.py

```diff
@@ -860,6 +861,7 @@ def __init__(self,
         :param end_trigger: when to end the optimization
         :param batch_size: training batch size
         """
+        warnings.warn("This is a deprecated method, you should use `create` method instead.")
```

Comment:
It is strange to say __init__ is deprecated?
What about changing to `You are recommended to use create....`

Reply:
Nice